internal/oc/release.go

```diff
@@ -336,15 +336,7 @@ func (r *release) Extract(log logrus.FieldLogger, releaseImage string, releaseIm
 // extractFromRelease returns the path to an openshift-baremetal-install binary extracted from
 // the referenced release image.
 func (r *release) extractFromRelease(log logrus.FieldLogger, releaseImage, cacheDir, pullSecret string, insecure bool, platformType models.PlatformType, icspFile string) (string, error) {
```

Comment:
Looks like `platformType` is not used any longer in this method

Reply:
That's correct, but I don't see any point backporting a [huge](https://github.com/openshift/assisted-service/pull/5334/commits/e73b9c5a68037f217110160d8912d54e583b631b) change to get rid of it all the way to 4.12.